### DISCUSSION TITLE
Persist operations on the explorer tab

### DISCUSSION THREAD

**Discussion by kdawgwilk** [apollographql/apollo-client-devtools#1533] [Ideas]
I would love to be able to persist the operations i have been working with on the Explorer tab so when i need to reload the page or come back later i don't have to rewrite the queries/mutations from scratch every time

**Comment by jerelmiller** [comment]
Hey @kdawgwilk! Good suggestion! The explorer is an iframe to our [sandbox](https://studio.apollographql.com/sandbox/explorer) so I'll talk to the team and see what we need to do to persist queries.

I can't guarantee anything anytime soon, but appreciate the suggestion!

**Reply by kdawgwilk** [reply]
~If I logged into the sandbox studio so i was authenticated would they be persisted automatically?~

It appears not, unfortunately
